lld/ELF/Writer.cpp

```diff
@@ -1900,6 +1932,18 @@ static void removeUnusedSyntheticSections() {
         auto *sec = cast<SyntheticSection>(s);
         if (sec->getParent() && sec->isNeeded())
           return false;
+        // Packed AArch64 AUTH relocs might be moved from .relr.auth.dyn to
```

Comment:
This is the only place "Packed AArch64 AUTH relocs" is mentioned. We can avoid this term. Just say
".relr.auth.dyn relocations may be moved to .rela dyn in finalizeAddressDependentContent, making .rela.dyn no longer empty. Conservatively keep .rela.dyn. .relr.auth.dyn can be made empty, but we would fail to remove it here."

Reply:
Fixed, thanks, see d32b8e32fc1c938e5a005997b832d215a308037d